### ISSUE TITLE
Stepping down as a maintainer

### ISSUE THREAD

**Issue by wafflespeanut** [wafflespeanut/paperclip#308]
Paperclip started off as an OpenAPI tooling library taking advantage of Rust's macros and type system. Initially, I only had client codegen in mind (which was more fun and challenging), but then [people liked the idea](https://github.com/actix/actix-web/issues/310) of autogenerating API docs at server side (which I thought wasn't a big deal), which is how paperclip gained more popularity. Back in 2019, I was actively following everything that happened in Rust community, but as time went on, new priorities took my attention. I thought I could spend more time in 2020, but I couldn't.

This was one of my favorite projects and I had a lot of hopes for implementing more features (OAPI3 support, plugin for rocket, tide, etc.), but given how the past few months have been going, it's time I admit that I won't be able to spend more time in maintaining this project. [I've tried my best](https://github.com/wafflespeanut/paperclip/blob/master/CHANGELOG.md) to implement all features from swagger (OAPI v2) for both actix-web plugin and client codegen and that was the 0.4.0 release. With that, I'll step down as a maintainer. I'll try to participate in discussions every now and then, but no promises.

As for the future maintainers of this project, I have some good candidates in mind: @platy and @dunnock already have privileges to this repo and I'm very happy about their contributions. If @tiagolobocastro, @MikailBag or @sfisol are willing to, then I can give privileges to them as well. Nominations are also welcome. I'll be active in [our Discord server](https://discord.gg/PPu4Dhj), so we can have a discussion there (or through mail, if you prefer).

A hearty "thank you" to all the contributors for supporting this project.

**Comment by tiagolobocastro** [comment]
First of all thank you for creating this project! Having you in the discord will still be of great help!
I don't have a lot of time but I'd be happy to maintain it with someone else, although not many people seem active here. Once we reach a certain point, maybe if we start creating plugins for other crates this will be more traction, one can but hope.
Thanks again :+1:

**Comment by platy** [comment]
Yes, thanks @wafflespeanut ! I have been really enjoying contributing to paperclip and it has been nice to work with you and the other contributors on it. I haven't been working on it recently and I don't think I will have enough time to take on a committed maintenance role - but I expect to contribute again. Thank you for being so welcoming!

**Comment by dunnock** [comment]
Thank you @wafflespeanut for a great project, it was spot on for our needs when we were looking at solution to generate swagger from our established actix APIs. Design idea is cool and it allows to keep boilerplate to bare minimum, which we like a lot. It also helps us to generate client libraries for 3 langs out of swagger specs. I will keep reviewing PRs but would be great to have more people involved in reviews as sometimes I have overloaded weeks.

**Comment by sfisol** [comment]
Thanks @wafflespeanut for your work!
I'll try to help as much as I can, though I don't know if my knowledge of all the paperclip internals is sufficient.

**Comment by ManevilleF** [comment]
Thank you for your work, I'll try to get into the core code to be able to contribute in the future

**Comment by Type1J** [comment]
@wafflespeanut Thanks for this great project!

@tiagolobocastro I don't have much time, but I like this project, and I believe that I'd end up rewriting everything in this project if I didn't use it instead.

Is there a project board?
What's the Discord server link?
What would you like me to do, if I were to be comaintainer with you?
